migrations/2024-07-31-063531_alter_customer_id_in_payouts/up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE payouts
+ALTER COLUMN customer_id
+DROP NOT NULL,
+ALTER COLUMN address_id
+DROP NOT NULL;
+
+ALTER TABLE payout_attempt
+DROP COLUMN IF EXISTS customer_id,
+DROP COLUMN IF EXISTS address_id;
```

Comment:
Add a comment stating that these need not be run on our hosted environments.

Reply:
Not needed, reverted the drop queries